Examples/test-suite/cpp11_template_explicit.i

```diff
@@ -4,6 +4,7 @@
 */
 %module cpp11_template_explicit
 
+/* Explicitely silence SWIG warning related to explicit templates */
```

Comment:
Comment typo "Explicitely".

(I thought I'd commented on this already - maybe your forced pushed lost it, maybe I failed to submit the comment, or maybe that was a different PR).